### ISSUE TITLE
Write the content for web development service

### ISSUE THREAD

**Issue by 1Copenut** [1Copenut/c3-eleventy#6]
There 4-6 core services that should be written. This issue will capture the copy for accessibility.

- [x] Write first draft
- [x] Proofread (ideally with outside help)
- [x] Add to Markdown file
